### DISCUSSION TITLE
deployment issue

### DISCUSSION THREAD

**Discussion by ShedrachMorris** [vercel/community#7147] [Help]
### Summary

after hosting my website, im getting a message 

404: NOT_FOUND
Code: NOT_FOUND
ID: cdg1::878hs-1717990905223-58540d9b280e

mean while everything is fine when i view the website locally.
anyone know why that is happening...?

### Steps to Reproduce

404: NOT_FOUND
Code: NOT_FOUND
ID: cdg1::878hs-1717990905223-58540d9b280e

**Comment by ontoast-simon-denny** [comment]
I have this exact same issue.

Deployment worked fine last week and website was up and running but tried to deploy a change this week and resulted in the website showing a blank page. I tried rolling back to the working version, but the same issue, a blank page.

I tried creating a new project and now get the same 404 error as you are experiencing so I expect something is wrong with vercel deployment?

**Reply by ontoast-simon-denny** [reply]
I've actually just added a missing vercel.json file and I no longer see the 404 error. I do however just see an empty screen even when I rollback to an older working version.

Have you added a vercel.json file yourself?

**Reply by Neurosissy** [reply]
I am experiencing the same issue. Adding the vercel.json file did not resolve it. This is how my backend looks: https://github.com/Neurosissy/neurosissy-website/tree/main/backend.
Here is my vercel.json configuration: https://github.com/Neurosissy/neurosissy-website/blob/main/backend/vercel.json.

The deployment ends up with the following error:

```
404: NOT_FOUND
Code: NOT_FOUND
ID: fra1::dnfqs-1718036647726-de36d4204dec
```

**P. S.** I imported the environment variables using a .env file, so everything should be set up correctly. What could be wrong here?

**Reply by ShedrachMorris** [reply]
i added the vercel.json file, it did no good

**Comment by amyegan** [comment]
Hey everyone. I've shared some tips for debugging 404 errors over in https://github.com/vercel/community/discussions/6980#discussioncomment-9705033

Some folks have also run into problems recently because of [breaking changes](https://turbo.build/blog/turbo-2-0#breaking-changes) between Turborepo v1 and v2. If your project uses Turborepo, then this guid might help you: https://turbo.build/repo/docs/crafting-your-repository/upgrading

If you can share a public [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), someone here might be able to recognize what's going wrong in your case.
